openapi.yaml

```diff
@@ -89,23 +89,35 @@ tags:
   - name: OSCAL Component Definition
     externalDocs:
       description: Find out more
-      url: https://pages.nist.gov/OSCAL/documentation/schema/implementation-layer/component/
+      url: https://pages.nist.gov/OSCAL/concepts/layer/implementation/component-definition/
   - name: OSCAL Profile
     externalDocs:
       description: Find out more
-      url: https://pages.nist.gov/OSCAL/documentation/schema/profile-layer/profile/
+      url: https://pages.nist.gov/OSCAL/concepts/layer/control/profile/
   - name: OSCAL System Security Plan
     externalDocs:
       description: Find out more
-      url: https://pages.nist.gov/OSCAL/documentation/schema/implementation-layer/ssp/
+      url: https://pages.nist.gov/OSCAL/concepts/layer/implementation/ssp/
   - name: OSCAL Catalog
     externalDocs:
       description: Find out more
-      url: https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/
+      url: https://pages.nist.gov/OSCAL/concepts/layer/control/catalog/
   - name: OSCAL Party
     externalDocs:
       description: Find out more
       url: https://pages.nist.gov/OSCAL
+  - name: OSCAL Plan of Action and Milestone
+    externalDocs:
+      description: Find out more
+      url: https://pages.nist.gov/OSCAL/concepts/layer/assessment/poam/
+  - name: OSCAL System Assessment Plan
+    externalDocs:
+      description: Find out more
+      url: https://pages.nist.gov/OSCAL/concepts/layer/assessment/assessment-plan/
+  - name: OSCAL System Assessment Result
```

Comment:
So I know this will take a little bit to change, but should we drop the "System"? 

https://github.com/EasyDynamics/OSCAL/tree/main/json/schema https://pages.nist.gov/OSCAL/reference/latest/complete/json-outline/ 

The files are just named `Assessment Plan` & `Assessment Result`. So I wonder if it is needed? That or we should change the name to `Security Assessment ...`

@kylelaker & @brian-easyd what do you think?

Reply:
I'd prefer that align with the NIST terminology here; and drop the first word. "Assessment Plan" and "Assessment Results" to match the documentation and schema.

Reply:
And when people casually refer to the meaning of SAP and SAR, it is a Security Assessment Plan and Security Assessment Result(s) in my neck of the woods. Do people actually call it System Assessment Plan in your experience?

Fun fact, and I agree with Dave Waltermire on this: we call them AP and AR and don't mention the S because, in 800-53 land, they can be security assessments, privacy assessments, or other. They aren't strictly always security assessments, and labeling them that way prejudices right from the start.

Also, hi, super excited to see this PR and the discussion here. :-)

Reply:
@aj-stein-nist Thanks! I think "System Assessment Plan" was some copying from "System Security Plan" and maybe a misunderstanding of it :D One thing we have noticed is that the [specification _does_ include Security](https://github.com/usnistgov/OSCAL/blob/main/src/metaschema/oscal_assessment-plan_metaschema.xml#L18) but we can open an issue upstream to discuss that if it'd be useful.

I definitely like the idea of not including the "S" to cover the fact it can be for more than one thing!

Reply:
@kylelaker @aj-stein-nist - In FedRAMP parlance (and most Agency implementations of FISMA) they are:
- Security Assessment Plan (SAP); and
- Security Assessment Report (SAR)  [NOTE: Not "result", but "report".]

"System" is not typically used.

That said, Dave and I debated the AP, AR name and he made the very valid point that while these are the correct terms for government use, OSCAL is more than just for the Federal government. He asserted that "Assessment Plan" and "Assessment Result" were more friendly to all frameworks.